runsc/boot/fs.go

```diff
@@ -765,6 +776,15 @@ func (c *containerMounter) getMountNameAndOptions(conf *Config, m specs.Mount) (
 		useOverlay bool
 	)
 
+	for _, opt := range m.Options {
+		// When options include either "bind" or "rbind", this behaves as bind mount.
+		// However, if mount type is equal to a filesystem supported on runsc, this prioritizes it over bind mount.
+		if (opt == "bind" || opt == "rbind") && !specutils.ContainsStr(supportedFilesystems, m.Type) {
```

Comment:
According to the spec, type can be ignored for bind mounts. So you don't need to check for `supportedFilesystems`.

> For bind mounts (when options include either bind or rbind), the type is a dummy, often "none"